NAMESPACE

```diff
@@ -3,6 +3,7 @@
 S3method(print,gt_tbl)
 export("%>%")
 export(adjust_luminance)
+export(as.tags.gt_tbl)
```

Comment:
This needs to be `S3method(as.tags,gt_tbl)`, there should be some way to convince roxygen to do that? @wch?

Reply:
Oh, adding `@method as.tags gt_tbl` does the trick. Not sure why it's necessary.

Reply:
Thanks for checking the NAMESPACE for this. This is now updated.

Reply:
I also reexported `htmltools::as.tags` for this to work (`htmltools` is in Imports).